docs/usage/SignatureVerification.md

```diff
@@ -0,0 +1,73 @@
+# Signature Verification
+
+Landscaper can verify the signature of a component version and only continue with the installation if the signature is valid and matches the content.
+
+> **IMPORTANT**: In the current state, the landscaper can only verify the signature and content when the installation is created/modified. Later controllers such as deployer will access the resources WITHOUT further verification. This implies that a potential malicious manipulation between creating the installation (and verifying the signature) and subsequent resource access can NOT be detected/prevented. Therefor, it is advisable to use this feature with a trusted repository only.
+
+## Requirement
+
+Verification only works when using OCM lib and OCM components. This is enabled as useOCMLib in the `landscaper-config.yaml`:
+```yaml
+useOCMLib: true
+```
+
+## Enable Verification of Component Versions
+
+### Landscaper Config Signature Verification Enforcement Policy
+
+In the landscaper config, a verification enforcement policy can be specified:
+```yaml
+useOCMLib: true
+signatureVerificationEnforcementPolicy: DoNotEnforce # DoNotEnforce(DEFAULT)|Enforce|Disabled
+```
+The following values are possible:
+- **DoNotEnforce** (DEFAULT): does not enforce a landscaper-wide policy. Each installation can specify a `spec.verification` key to enable the verification for it.
+- **Enforce**: enforces all installations to be verified and therefore all installation need the `spec.verification` key with provided verification options.
+- **Disabled**: explicitly disable all verifications for all installations even if a installations provides verification information with `spec.verification` key. **It is advised to use this option for debugging only.**
+
+## Configure Verification
+
+Necessary verification information is provided in the installation and the context.
+In the installation, `spec.verification.signatureName` needs to be set to the name of the signature in the component version:
```

Comment:
Isnt't it the name of the signature in the context?

Reply:
yes, in the context we have a map of signatureName: {} that allows multiple signature names to be defined in the context. For the installation, the user must pick which signatureName should be chosen.

Reply:
But you wrote "needs to be set to the name of the signature in the component version" instead of "needs to be set to the name of the signature in the context" :-)

Reply:
Now I understand, the name of the signature in the component version == name of the signature in the context == name of the signature in the installation.